README.md

```diff
@@ -46,6 +46,34 @@ Usage
 Unless special requirements disallow the usage of asyncio, by far the easiest
 way to use nio is using the asyncio layer:
 
+
+Sending
+-------
+
+```python
+import asyncio
+from nio import AsyncClient
+
+async def main():
+    client = AsyncClient("https://example.org", "@alice:example.org")
+    
+    await client.login("hunter1")
+    await client.room_send(
+        "ROOM_ID_HERE:example.org",
+        "m.room.message",
+        {
```

Comment:
This can be improved by having the argument names, e.g. `room_id="!test:example.org"`

Reply:
I don't 100% know what you mean but I'll change it to your example

Reply:
Like this:
```
room_id="!test:example.org",
message_type="m.room.message"
content={
    ...
```
It helps users knowing what the arguments are called and what they are for.

Reply:
ah, now I get what you mean, I completely looked at the wrong thing